app/models/deliveryman.rb

```diff
@@ -0,0 +1,7 @@
+class Deliveryman < ActiveRecord::Base
+  
+  validates :name, presence: true
+  validates :phone, presence: true
+  validates :bank_info, presence: true
```

Comment:
We should also validate the format of phone numbers and bank information. Not something that needs to be done right now though.